public/js/homepage.js

```diff
@@ -0,0 +1,29 @@
+// btn-hence working on service button carousel
+
+$('.btn-hence').owlCarousel({
```

Comment:
why not using a specific name for this slide
do not use css class for js identifier 
give a new meaning full class name for js identifier and use that in js script

Reply:
i provide meaning full class name